glide.yaml

```diff
@@ -11,7 +11,10 @@ import:
 - package: k8s.io/kube-aggregator
   version: kubernetes-1.12.5
 - package: k8s.io/apiextensions-apiserver
-  version: kubernetes-1.12.5
+  repo: https://github.com/openshift/kubernetes-apiextensions-apiserver
+  version: origin-4.0-kubernetes-1.12.4 # for nullable PR, instead of kubernetes-1.12.5
+- package: sigs.k8s.io/controller-tools
+  repo: https://github.com/openshift/kubernetes-sigs-controller-tools
```

Comment:
What do we need this for? If it's for the CRD-generator, shouldn't it be a part of library-go deps if it provides the command?

Reply:
library-go just has a script which calls the necessary components of controller-tools to generate openapi validations. glide strips transitive dependencies so we need to import the fork to be able to run the script here

Reply:
Ah, I see, thanks!